src/nunavut/_utilities.py

```diff
@@ -93,7 +93,9 @@ def test_truth(cls, ynd_value: "YesNoDefault", default_value: bool) -> bool:
     DEFAULT = 2
 
 
-def iter_package_resources(pkg_name: str, *suffix_filters: str) -> Generator[pathlib.Path, None, None]:
+def iter_package_resources(
+        pkg_name: str, 
+        *suffix_filters: str) -> Generator[tuple[pathlib.Path, pathlib.Path], None, None]:
```

Comment:
Consider setting up Black locally to avoid complaints from the CI system (it will complain about this fragment). I found autoformatting on-save to be super convenient.